BEIMA.Client/src/App.js

```diff
@@ -31,6 +32,7 @@ const App = () => {
           <Route path="devices/:id" element={<DevicePage/>}/>
           <Route path="deviceTypes" element={<DeviceTypesPage/>}/>  
           <Route path="deviceTypes/:typeId" element={<DeviceTypePage/>}/>
+          <Route path="buildings/addBuilding" element={<AddBuildingPage/>}/>  
```

Comment:
Should we add a link on the navbar for this page?

Reply:
I'm going to add one for buildings list, and then from that page there will be an "Add Building" button that will lead to this page

Reply:
But we can add it to the nav bar, I just don't want to add too much, and the buildings will probably be something that isn't modified a whole lot, just to get the current buildings added.